routing/routing_tests/road_access_test.cpp

```diff
@@ -13,7 +14,7 @@
 using namespace routing;
 using namespace std;
 
-namespace
+namespace routing_test
```

Comment:
Why? Keep anonymous namespace. If you need something from routing_tests just add a using namespace declaration.